include/motor-control/core/motor_interrupt_handler.hpp

```diff
@@ -37,45 +37,85 @@ namespace motor_handler {
  * GenericQueue -> A FreeRTOS queue of any shape.
  */
 
+constexpr const int clk_frequency = 85000000 / (5001 * 2);
```

Comment:
I wonder if this could become part of the motor config? that's created in the right place (in the task root) and has similar hardware config stuff